packages/components/src/menu-items-choice/types.ts

```diff
@@ -38,6 +38,10 @@ export type MenuItemChoice = {
 	 * Unique value for choice.
 	 */
 	value: string;
+	/**
+	 * Unique value for choice.
+	 */
+	disabled?: boolean;
```

Comment:
I think this could use some love. 

On one hand, it's odd that the `MenuItemChoice` is typed but the `MenuItem` isn't, and that creates some of the problems - the `disabled` type should be declared as part of the `MenuItem` type ideally, but let's say that's for another PR.

However, in this PR I think we can still make things a bit more precise. The `MenuItemChoice` uses a `MenuItem` component under the hood, and `MenuItem` component uses a `Button` component under the hood. Given that all additional `MenuItemChoice` props are added as `buttonProps` to the `MenuItem` and therefore passed down to the underlying `Button`, we likely have to provide a partial `ButtonProps` type, or worst case scenario, borrow the `disabled` type from `Button.ButtonProps[ 'disabled' ]`.

Reply:
Imported type definition eee6284b93fed270a20e9e6d7b5383dc0f2c58b6